### ISSUE TITLE
Add CookDSP credits and copyright notices where appropriate

### ISSUE THREAD

**Issue by lossius** [ambisonictoolkit/atk-reaper#42]

**Comment by joslloand** [comment]
@lossius 

The [JSFX - CookDSP Library](http://ajaxsoundstudio.com/software/cookdsp/), [GitHub here](https://github.com/belangeo/cookdsp), looks like a very useful and interesting find. Among other things, using it should make certain tasks much easier. I'm just browsing the documentation now.

I wasn't seeing a use notification, but see it here, now:

```
/////////////////////////////////////////////////////////////////
// Copyright (c) - 2014 - Olivier Belanger                     //
// Permission to use, copy, modify, and/or distribute this     //
// software for any purpose with or without fee is hereby      //
// granted, provided that the above copyright notice and this  //
// permission notice appear in all copies. CookDSP IS PROVIDED //
// "AS IS", WITHOUT WARRANTY OF ANY KIND FROM THE AUTHOR.      //
/////////////////////////////////////////////////////////////////
```

Have you been in touch with Belanger?

**Comment by lossius** [comment]
Yes, CookDSP is compatible, and I have made a series of other plugins using it. It is really nice and elegantly programmed, if JSFX can ever be elegant...

Further down the road I might refactor parts of the library code in ATK for Reaper to be object oriented in the same way as CookDSP. This might also make it easier to implement pseudoInverse().

Another benefit that I see the potential for are additiuonal ambisonics plugins that will be beyond the scope of ATK itself, but use ATK and CookDSP in order to e.g. create B-format granulation engines with individual stocastic rotation/tilt/tumbling of each grain.

I have a playground for these kind of js plugins here:

https://github.com/lossius/js-plugins

It's not at all complete or coherent, and is continuously altered and changed depending on my own whims and needs for projects, but for the time being it serves as a playground for starting to conceptualise this. I have not announced this repo anywhere, as I want to maintain the possibility of breaking backwards compatibility whenever I feel for it, but recently I implemented a 4 channel EQ that turned out to be crucial for adjusting to the acoustics of the space at the museum where I did a new installation two weeks ago in Sogn & Fjordane.

There's a screenshot of the EQ here:

http://trondlossius.no/articles/1277-further-refinement-of-4-channel-equalizer
